extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -64,7 +66,7 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
 
   // TODO -> Two clients sucks, but its better than 1000.
   // TODO -> We'll need to merge auth later.
-  const qidoDicomWebClient = new api.DICOMwebClient(qidoConfig);
+  const qidoDicomWebClient = new StaticWadoClient(qidoConfig);
```

Comment:
I understand `StaticWadoClient` extends the `api.DICOMWebClient` and returns early if no staticWado config; however, I would still prefer (for readability) simpler if statement that chooses the client based on `staticWado` like 

`const qidoDicomWebClient = staticWado ? new StaticWadoClient(...) : api.DICOMwebClient(...)`